services/storage/signed-uploads/webpack.config.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Not required ;)

Reply:
I have just copied the configurations as I found them in other places.